en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>latency</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Represents Latency(95th percentile) time in milliseconds(ms).
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### Subscriber
+
+Represents API Subscriber details.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>name</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API subscriber name.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### SuccessAPIUsageByApplication
+
+Represents successful(2xx response code) usages of an API by an application.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>apiName</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>apiVersion</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API version.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>apiCreatorTenantDomain</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Tenant domain of the API creator.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationId</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationName</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationOwner</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Application name owner.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td></td>
+</tr>
+</tbody>
+</table>
+
+### SuccessSummary
+
+Provides successful(2xx response code) usage summary across all the APIs.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#successvalue">SuccessValue</a>]!</td>
+<td>
+
+List containing successful usage values. Returns an empty array if no data available.
```

Comment:
Start the first sentence with "A" or "The" as relevant, and add "is" after "data".